src/content/en/updates/posts/2016/02/devtools-digest-supercharged-remote-debugging.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: updates/post
+title: "Supercharged Remote Rebugging, Class Toggles and our own late night show?!"
+published_on: 2016-02-25
+updated_on: 2016-02-25
+authors:
+  - pbakaus
+tags:
+  - devtools
+  - digest
+  - update
+description: "Learn all about the revamped “Inspect Devices” UI, toggle classes easily in the now-fixed style panel and watch the pilot of DevTools Tonight."
+featured_image: /web/updates/images/2016/02/devtools-digest/devtools-tonight.jpg
+---
+
+<p class="intro">Learn all about the revamped “Inspect Devices” UI, toggle classes easily in the now-fixed style panel and watch the pilot of DevTools Tonight.</p>
```

Comment:
We usually bold (embolden?) UI elements... So **Inspect Devices** rather than "Inspect Devices"